glue/core/roi.py

```diff
@@ -193,6 +223,15 @@ def move_to(self, x, y):
         self.ymin += dy
         self.ymax += dy
 
+    def rotate_to(self, theta):
+        """ Rotate anticlockwise around center to position angle theta (radian) """
+        self.theta = 0 if theta is None else theta
+        if np.isclose(self.theta % np.pi, 0.0, atol=1e-9):
+            self.rotation = np.identity(2)
+        else:
+            self.rotation = rotation(self.theta)
+        self.invrot = self.rotation * [[1, -1], [-1, 1]]
```

Comment:
I don't think we should store three different variables that are for the same thing in case they get out of sync. Perhaps we should just store theta and then have the matrices be properties (and it might be premature optimization to worry about caching these)

Reply:
There is one issue with using this on degenerate (1D) polygons, where `self.centroid()` returns [NaN, NaN] (mathematically correct, I guess, since it relates to the polygon area. Not sure if that should be considered;
could simply set it to the mean for those cases. That might actually be consistent enough.